### DISCUSSION TITLE
do I need to have subscribed to some content?

### DISCUSSION THREAD

**Discussion by Bruneraaa** [UltimaHoarder/UltimaScraper#1623] [Q&A]
in order for me to download the contents do I need to have one already signed? Or can I download without having signed up?

**Comment by kavinklain** [comment]
yes, you have to be subscribed. This scraper does not bypass any paywall. It will download all posts' contents of the model that you have subscribed to. It will not scrape posts behind a paywall (ie needs you to pay another separate amount of $ to view).

**Reply by Bruneraaa** [reply]
thanks dude
